[BUG] Crashing when using external repository

**Describe the bug**
I use nextcloud to synchronize my .password-store directory to android, then configure Password Store to use an external repo and point to the .password-store folder in my nextcloud directory.  I also use Open Keychain as the OpenPGP provider, but I don't think that's an issue here.

With the above configuration, Password Store immediately crashes every time I open it.

This configuration has worked previously for me on several versions of android and on several phones, but has consistently crashed as described since moving to a Pixel 3a with Android 10, so I suspect it's somehow related to Android 10.

**To Reproduce**
Steps to reproduce the behavior:
1. Clear storage data to start from scratch and open Password Store
2. Select Use Local Directory
3. Select SD-Card, then OK to choose where to store passwords
4. Select the already synchronized .password-store folder from nextcloud (/Android/media/com.nextcloud.client/nextcloud/user@domain/keys/.password-store)
5. When prompted to allow access to that folder, select Allow
6. You will be returned to the Settings screen.  Return to main page.
7. Observe crash.
8. Try to launch Password Store again; observe crash; repeat

**Expected behavior**
I'd expect Password Store to not crash

**Device information (please complete the following information):**
 - Device: Pixel 3a
 - OS: stock rom Android 10 / November 5, 2019 security update applied
 - App version: 1.3.3 (from f-droid)

Pasting below.  I've never used logcat before, so if this is not what you're looking for would appreciate a little extra guidance.

```
12-15 02:36:39.481  1280  6723 I ActivityTaskManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.zeapo.pwdstore/.PasswordStore bnds=[0,364][216,671]} from uid 10237
12-15 02:36:36.245   838  2211 I chatty  : uid=1010 wifi_ext@1.0-se identical 4 lines
12-15 02:36:38.251   838  2211 I WifiHAL : event received NL80211_CMD_VENDOR, vendor_id = 0x1374, subcmd = 0xd
12-15 02:36:39.489   829   829 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 1
12-15 02:36:39.498  1280  1351 E system_server: Invalid ID 0x00000000.
12-15 02:36:39.521   798   798 D Zygote  : Forked child process 8831
12-15 02:36:39.525  1280  1368 I ActivityManager: Start proc 8831:com.zeapo.pwdstore/u0a254 for activity {com.zeapo.pwdstore/com.zeapo.pwdstore.PasswordStore}
12-15 02:36:39.535  8831  8831 E .zeapo.pwdstor: Not starting debugger since process cannot load the jdwp agent.
12-15 02:36:39.551  1280  6723 W ActivityTaskManager: Request to remove task ignored for non-existent task 970
12-15 02:36:39.560   767   767 I netd    : firewallSetUidRule(2, 10254, 1) <0.03ms>
12-15 02:36:39.561   767   767 I netd    : bandwidthAddNiceApp(10254) <0.05ms>
12-15 02:36:39.653  8831  8831 D AndroidRuntime: Shutting down VM
12-15 02:36:39.654  8831  8831 E AndroidRuntime: FATAL EXCEPTION: main
12-15 02:36:39.654  8831  8831 E AndroidRuntime: Process: com.zeapo.pwdstore, PID: 8831
12-15 02:36:39.654  8831  8831 E AndroidRuntime: java.lang.RuntimeException: Unable to resume activity {com.zeapo.pwdstore/com.zeapo.pwdstore.PasswordStore}: java.lang.NullPointerException: Attempt to get length of null array
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4205)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:4237)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:52)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:176)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:97)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2016)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:107)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:214)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7356)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
12-15 02:36:39.654  8831  8831 E AndroidRuntime: Caused by: java.lang.NullPointerException: Attempt to get length of null array
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at com.zeapo.pwdstore.utils.PasswordRepository.initialize(PasswordRepository.java:143)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at com.zeapo.pwdstore.PasswordStore.checkLocalRepository(PasswordStore.java:358)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at com.zeapo.pwdstore.PasswordStore.onResume(PasswordStore.java:156)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.Instrumentation.callActivityOnResume(Instrumentation.java:1453)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.Activity.performResume(Activity.java:7939)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        at android.app.ActivityThread.performResumeActivity(ActivityThread.java:4195)
12-15 02:36:39.654  8831  8831 E AndroidRuntime:        ... 11 more
12-15 02:36:39.657  1280  8855 I DropBoxManagerService: add tag=data_app_crash isTagEnabled=true flags=0x2
12-15 02:36:39.657  1280  6723 W ActivityTaskManager:   Force finishing activity com.zeapo.pwdstore/.PasswordStore
12-15 02:36:39.663  1280  1367 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to com.google.android.gms/.stats.service.DropBoxEntryAddedReceiver
12-15 02:36:39.663  1280  1367 W BroadcastQueue: Background execution not allowed: receiving Intent { act=android.intent.action.DROPBOX_ENTRY_ADDED flg=0x10 (has extras) } to com.google.android.gms/.chimera.GmsIntentOperationService$PersistentTrustedReceiver
12-15 02:36:39.663  8831  8831 I Process : Sending signal. PID: 8831 SIG: 9
12-15 02:36:39.671  1280  2977 I ActivityManager: Process com.zeapo.pwdstore (pid 8831) has died: vis+99 TOP
12-15 02:36:39.672  1280  1369 I libprocessgroup: Successfully killed process cgroup uid 10254 pid 8831 in 0ms
12-15 02:36:39.673   798   798 I Zygote  : Process 8831 exited due to signal 9 (Killed)
12-15 02:36:39.689   767   767 I netd    : bandwidthRemoveNiceApp(10254) <0.04ms>
12-15 02:36:39.715   829   829 D android.hardware.power@1.3-service.pixel-libperfmgr: LAUNCH: 0
12-15 02:36:40.161  1280  1349 W ActivityTaskManager: Activity top resumed state loss timeout for ActivityRecord{e98b094 u0 com.zeapo.pwdstore/.PasswordStore t-1 f}
12-15 02:36:40.253   838  2211 I WifiHAL : event received NL80211_CMD_VENDOR, vendor_id = 0x1374, subcmd = 0xd
```

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -19,6 +19,7 @@
         android:allowBackup="false"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
+        android:requestLegacyExternalStorage="true"
         android:supportsRtl="true"
         android:theme="@style/AppTheme"
         tools:ignore="GoogleAppIndexingWarning">

